include/ifc/abstract-sgraph.hxx

```diff
@@ -2891,20 +2891,15 @@ namespace ifc {
             using ValueType = T;
         };
 
-// Note: VC and the build settings do not want to see idiomatic C++ by default.
-#pragma warning(push)
-#pragma warning(disable : 4624)
+
         struct IntegerLiteral : constant_traits<uint64_t, LiteralSort::Integer> {};
 
-#pragma pack(4)
```

Comment:
Was this packing necessary at all? `FloatingPointLiteral` is the only reference to this and even that type has no references to it.

Reply:
Yes, they were found to be necessary a few years ago.  Please leave them in for now, and let's conduct a new investigation in a separate PR.

Reply:
Changed the pragma to a macro that works for MSVC, GCC and Clang. The size of the struct is also verified.

Reply:
Ugh, macros.  But, I see we have nothing better for now.  OK.